src/getPdfPagesCount.ts

```diff
@@ -1,9 +1,9 @@
-import PdfMetadata from "./pdfMetadata";
+import { PdfMetadata } from "./types";
 import { pdfMetadata } from "./getPdfMetadata";
 
 // Getting the .pdf total pages count
-export const getPdfPagesCount = async(): Promise<number> => {  
-  const metadata: PdfMetadata = await pdfMetadata();
-  const pageCount = metadata.pages;
-  return pageCount;
-}
\ No newline at end of file
+export const getPdfPagesCount = async () => {
```

Comment:
TS can infer the return type of this async function. No need to specify it.

As a rule of thumb, Let TS infer as much as it can and only add types when it is struggling